### PR TITLE
Bugs, Feature

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy [hunk 1]
```diff
@@ -262,6 +262,18 @@ class SubscriptionController extends AbstractDebugController {
...
         Set<IssueEntitlement> entitlements = IssueEntitlement.executeQuery("select ie " + base_qry, qry_params)
 
+        if(params.kbartPreselect) {
+            CommonsMultipartFile kbartFile = params.kbartPreselect
+            InputStream stream = kbartFile.getInputStream()
+            List issueEntitlements = entitlements.toList()
+
+            subscriptionService.issueEntitlementEnrichment(stream, issueEntitlements, (params.uploadCoverageDates == 'on'), (params.uploadPriceInfo == 'on'))
+
+            params.remove("kbartPreselct")
```
Comment (klodav) [inline app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy:272]: kbartPreselect

File: app/grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -2022,6 +2022,10 @@ subscription.details.addEntitlements.noPackagesYetAdded = You have not added a p
 subscription.details.createEntitlementGroup.label = Create Title Group
 subscription.details.manageEntitlementGroup.label = Manage Title Group
 
+subscription.details.issueEntitlementEnrichment.label = Title-Enrichment
```
Comment (croqueGrec09) [inline app/grails-app/i18n/messages.properties:2025]: Title enrichment

File: app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy (hunk 1 above)
Comment (croqueGrec09) [reply app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy:272]: Maybe that was even the reason for the overwrite?

File: app/web-app/setup/RefdataValue.csv [hunk 3]
```diff
@@ -1141,6 +1141,7 @@ user.setting.dashboard.tab,PendingChanges,Änderungen,Changes,,
 user.setting.dashboard.tab,AcceptedChanges,Benachrichtigungen,Notifications,,
 user.setting.dashboard.tab,Due Dates,Fällige Termine,Due Dates,,
 user.setting.dashboard.tab,Tasks,Aufgaben,Tasks,,
+user.setting.dashboard.tab,Surveys,Umfragen,Survey,,
```
Comment (klodav) [inline app/web-app/setup/RefdataValue.csv:1144]: Surveys

File: app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy (hunk 1 above)
Comment (Moetez16) [reply app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy:272]: No. What is the problem.

**Pull request merged**